config/puma.rb

```diff
@@ -47,10 +49,11 @@
 # or connections that may have been created at application boot, as Ruby
 # cannot share connections between processes.
 #
-# on_worker_boot do
-#   ActiveRecord::Base.establish_connection if defined?(ActiveRecord)
-# end
-#
+on_worker_boot do
+  ActiveRecord::Base.establish_connection if defined?(ActiveRecord)
+
+  CertificateStore.instance.load_certs!
```

Comment:
this duplicates a call in `config/initializers/certificate_store.rb` , but I couldn't figure out how to detect "puma" mode and skip that one, at the end of the day, I don't think it's harmful to have both so it's prob fine